entity-service-change-event-generator/src/main/java/org/hypertrace/entity/service/change/event/impl/EntityChangeEventGeneratorImpl.java

```diff
@@ -147,6 +172,67 @@ private void sendUpdateNotification(
     }
   }
 
+  private boolean shouldSendNotification(Entity prevEntity, Entity currEntity) {
+    boolean isAttributesChanged = false, isCoreEntityChanged = false;
+    for (FieldDescriptor fieldDescriptor : Entity.getDescriptor().getFields()) {
```

Comment:
Sorry, why are we messing with the proto here? The fields inside the attributes map should be all we care about.